cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-item-returned.cy.js

```diff
@@ -0,0 +1,295 @@
+          Checkout.checkoutItemViaApi({
+            id: uuid(),
+            itemBarcode: item.barcode,
+            loanDate: moment.utc().format(),
+            servicePointId: testData.userServicePoint.id,
+            userBarcode: userData.barcode,
+          });
+        });
+
+        UserLoans.getUserLoansIdApi(userData.userId).then((response) => {
+          const resp = response.loans;
+          cy.log(resp);
+          resp.forEach(({ id, item }) => {
+            if (item.title.includes('InstanceForDeclareLost')) {
+              UserLoans.declareLoanLostByApi({ servicePointId: testData.userServicePoint.id }, id);
+            }
+          });
+        });
+
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    cy.get('@items').each((item) => {
+      CheckInActions.checkinItemViaApi({
+        itemBarcode: item.barcode,
+        servicePointId: testData.userServicePoint.id,
+        checkInDate: new Date().toISOString(),
+      });
+    });
+    PaymentMethods.deleteViaApi(testData.paymentMethodId);
+    UsersOwners.deleteViaApi(owner.data.id);
+    cy.deleteLoanPolicy(loanPolicyBody.id);
+    LostItemFeePolicy.deleteViaApi(lostItemFeePolicyBody.id);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+
+    cy.getToken(userData.username, userData.password);
+    UserLoans.updateTimerForAgedToLost('reset');
+    cy.getAdminToken();
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    Conditions.resetConditionViaApi('cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a', 'Maximum outstanding fee/fine balance');
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    CirculationRules.deleteRuleViaApi(originalCirculationRules);
+    cy.deleteLoanType(testData.loanTypeId);
+  });
+  it(
+    'C350651 Verify automated patron block "Maximum outstanding fee/fine balance" removed after lost item returned (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      cy.visit(SettingsMenu.conditionsPath);
+      Conditions.waitLoading();
+      Conditions.select('Maximum outstanding fee/fine balance');
+      Conditions.setConditionState(blockMessage);
+      cy.visit(SettingsMenu.limitsPath);
+      Limits.selectGroup(patronGroup.name);
+      Limits.setMaximumOutstandingFeeFineBalance('624');
+      cy.wait(230000);
```

Comment:
please explain why do you need this waiter

Reply:
done